content/applications/finance/fiscal_localizations/colombia.rst

```diff
@@ -37,208 +34,376 @@ localization:
-     - Point of Sale
-   * - :guilabel:`Colombian - Accounting Reports`
-     - `l10n_co_reports`
-     - Colombian reports
+     - Includes Point of Sale Receipt for Colombian Localization.
 
-Configure credentials for Carvajal web service
-----------------------------------------------
+.. note::
+   When a database is created from scratch selecting :guilabel:`Colombia` as the country, Odoo
+   automatically installs the base modules *Colombia - Accounting* and *Colombia - Accounting
+   Reports*.
 
-Once the modules are installed, the user credentials need to be configured in order to connect with
-Carvajal Web Service. First, navigate to :menuselection:`Accounting --> Configuration --> Settings`
-and look for the :guilabel:`Colombian Electronic Invoice` section. Then, fill in the required
-configuration information provided by Carvajal.
+Company configuration
+---------------------
 
-.. image:: colombia/carvajal-credential-config.png
-   :alt: Configure credentials for Carvajal web service in Odoo.
+To configure your company information, go to the :menuselection:`Contacts` app and search for your
+company. Alternatively, activate :ref:`developer mode <developer-mode>` and navigate to
+:menuselection:`General Setting --> Company --> Update Info --> Contact`. Then, edit the **contact
+form** to configure the following information:
 
-Check the :guilabel:`Test mode` checkbox to connect with the Carvajal testing environment. This
-allows users to test the complete workflow and integration with the :abbr:`CEN (Centro Electrónico
-de Negocios)` Financiero portal, which is accessible here:
+#. General data
 
-- `CTS (Carvajal T&S) <https://cenflab.cen.biz/site/>`_.
-- `CSC (Carvajal Servicios de Comunicación) <https://web-stage.facturacarvajal.com/>`_.
+   - :guilabel:`Name`.
+   - :guilabel:`Address`: Including :guilabel:`City`, :guilabel:`Department` and :guilabel:`Zip
+     Code`.
+   - :guilabel:`Tax ID`: When it is a `NIT`, it must have the *verification digit* at the end of the
+     ID followed by a hyphen (`-`).
 
-:abbr:`CSC (Carvajal Servicios de Comunicación)` is the default for new databases.
+   .. image:: colombia/company-configuration.png
+      :alt: Configure your company contact form in Odoo.
 
-Once Odoo and Carvajal are fully configured and ready for production, the testing environment can be
-disabled by unchecking the :guilabel:`Test mode` checkbox.
+#. :guilabel:`Fiscal Information`
 
-Configure report data
----------------------
+   In the :guilabel:`Sales & Purchase` tab, configure the following information:
+
+   - :guilabel:`Fiscal Position`: Determines the taxes/accounts used for this contract.
+   - :guilabel:`Tipo de Representación`.
+   - :guilabel:`Tipo Establecimiento`.
+   - :guilabel:`Obligaciones y Responsabilidades`.
+   - :guilabel:`Usuario Aduanero`.
+   - :guilabel:`Gran Contribuyente`.
+   - :guilabel:`Fiscal Regimen`.
+   - :guilabel:`Commercial Name`.
+
+   .. image:: colombia/fiscal-information.png
```

Comment:
The screenshot is not useful in its current state as it duplicates the info provided above.
Consider removing it, or if you think it's helpful, fill out the fields with a believable example that could help users.

NB: As a general rule, screenshots should give additional information / illustrate something extra or unexpected (e.g., a configuration outside of Odoo, an example of how something looks like after configuration...). In this case, I don't think any screenshot is needed.

Let me reiterate: screenshots are harder to maintain than text; they are not searchable; they are not very visible on portable devices... and most people don't take "good-looking" screenshots, with issues such as being blurry, poorly cropped, taken using different settings (meaning they don't look the same).

Could you revise the whole PR accordingly, as this comment definitely applies to other screenshots here?